server/lib/hyperwatch.js

```diff
@@ -74,7 +74,8 @@ const load = async app => {
         log = log.deleteIn(['request', 'headers', 'cookie']);
 
         if (req.body && req.body.query && req.body.variables) {
-          log = log.set('graphql', req.body);
+          const bodyStr = redactSensitiveFields(req.body);
```

Comment:
Could we have `redactSensitiveFields` return an object and not a string?

Reply:
Yes, do you think is preferred? I was on the fend for this as the Sentry method takes a serialized string and the hyperwatch an object.
This library used to redact always return the serialized version.